backend/wmg/api/wmg-api-v2.yml

```diff
@@ -0,0 +1,821 @@
+servers:
+  - description: Local
+    url: /
+  - description: Production environment
+    url: https://api.cellxgene.cziscience.com/
+  - description: Development environment
+    url: https://api.dev.single-cell.czi.technology/
+  - description: Staging environment
+    url: https://api.staging.single-cell.czi.technology/
+
+paths:
+  /primary_filter_dimensions:
+    get:
+      summary: Returns the ontology terms for organism and tissue type that can be used when specifying a WMG query.
+      description: >-
+      tags:
+        - wmg
+      operationId: backend.wmg.api.v2.primary_filter_dimensions
+      parameters: []
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  snapshot_id:
+                    $ref: "#/components/schemas/wmg_snapshot_id"
+                  organism_terms:
+                    $ref: "#/components/schemas/wmg_ontology_term_id_label_list"
+                  tissue_terms:
+                    # TODO: specify further: https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/1967
+                    # {"organism_ontology_term_id_0": [ {"tissue_ontology_term_id_0": "tissue_ontology_term_id_0_label"}],}
+                    type: object
+                  gene_terms:
+                    # TODO: specify further: https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/1967
+                    # {"organism_ontology_term_id_0": [ {"gene_ontology_term_id_0": "gene_ontology_term_id_0_label"}],}
+                    type: object
+        "404":
+          $ref: "#/components/responses/404"
+  /query:
+    post:
+      tags:
+        - wmg
+      operationId: backend.wmg.api.v2.query
+      parameters: []
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                filter:
+                  type: object
+                  required:
+                    - gene_ontology_term_ids
+                    - organism_ontology_term_id
+                  properties:
+                    gene_ontology_term_ids:
+                      $ref: "#/components/schemas/wmg_ontology_term_id_list"
+                    organism_ontology_term_id:
+                      type: string
+                    dataset_ids:
+                      type: array
+                      items:
+                        type: string
+                        format: uuid
+                    disease_ontology_term_ids:
+                      $ref: "#/components/schemas/wmg_ontology_term_id_list"
+                    sex_ontology_term_ids:
+                      $ref: "#/components/schemas/wmg_ontology_term_id_list"
+                    development_ontology_stage_term_ids:
```

Comment:
Looks like there's a typo here, not sure why that's only becoming an issue for me now though.

Reply:
```suggestion
                    development_stage_ontology_term_ids:
```